packages/bundler-plugin-core/src/sentry/releasePipeline.ts

```diff
@@ -86,40 +86,31 @@ export async function finalizeRelease(options: InternalOptions, ctx: BuildContex
   span?.finish();
 }
 
-export async function cleanArtifacts(options: InternalOptions, ctx: BuildContext): Promise<string> {
+export async function cleanArtifacts(options: InternalOptions, ctx: BuildContext): Promise<void> {
   const span = addSpanToTransaction(ctx, "function.plugin.clean_artifacts");
 
   if (options.cleanArtifacts) {
     // TODO: pull these checks out of here and simplify them
     if (options.authToken === undefined) {
       ctx.logger.warn('Missing "authToken" option. Will not clean existing artifacts.');
-      return Promise.resolve("nothing to do here");
+      return;
```

Comment:
l: really feels like this should be abstracted out - perhaps into an array that gets iterated on (we can then even validate this with types)